scheduler/scheduler/main.go

```
@@ -45,50 +44,39 @@ func main() {
 	}
 	log.SetLevel(level)
 
-	configText, err := jsonconfig.GetConfigText(*configFlag, config.Asset)
+	schedulerJSONConfigs, err := config.GetSchedulerConfigs(*configFlag)
 	if err != nil {
-		log.Fatal(err)
+		panic(fmt.Errorf("error parsing schedule server config.  Scheduler not started. %s", err))
```

why panic instead of log.Fatal?

---

changed back to log.Fatal()